go/vt/wrangler/reparent.go

```diff
@@ -216,6 +216,10 @@ func (wr *Wrangler) initShardMasterLocked(ctx context.Context, ev *events.Repare
 		return err
 	}
 
+	// Create a cancelable context for the following RPCs.
+	// If error conditions happen, we can cancel all outgoing RPCs.
+	journalCtx, journalCancel := context.WithCancel(ctx)
```

Comment:
You're using the context for changing the slaves as well.

Maybe give it a more generic name?